policy-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/policy/kafka/KafkaContractTest.java

```diff
@@ -277,41 +292,50 @@ void testNoSuchPolicyExceptionIsThrown() {
         // All messages have a correct Token in the header
         assertAll("Headers have correct token",
                 () -> assertThat(results)
+                        .as("Check the size of the returned results")
                         .hasSize(2),
 
                 () -> assertThat(results)
                         .allSatisfy(result ->
```

Comment:
allSatisfy is not needed here

Reply:
This is making sure that all the values within `results` list have the expected bytes for the token.